swift/Samples/SplitScreenContainer/SnapshotTests/SplitScreenContainerScreenSnapshotTests.swift

```diff
@@ -12,13 +11,13 @@ class SplitScreenContainerScreenSnapshotTests: FBSnapshotTestCase {
         folderName = "SplitScreenContainerScreenSnapshotTests"
         fileNameOptions = [.device]
     }
-    
+
     func test_splitRatio() {
-        let ratios: [String : CGFloat] = [
-            "third" : .third,
```

Comment:
this is interesting, not strongly opposed but I feel like I normally write this with a space